src/sentry/services/hybrid_cloud/organization/model.py

```diff
@@ -209,7 +209,7 @@ class RpcUserInviteContext(RpcUserOrganizationContext):
     member state of the invite if none such exists.
     """
 
-    invite_organization_member_id: int = 0
+    invite_organization_member_id: Optional[int] = 0
```

Comment:
the only caller passes `int | None` here: https://github.com/getsentry/sentry/blob/4e15eceb19432a2d976d2a3a8f924f90a9fd4632/src/sentry/services/hybrid_cloud/organization/impl.py#L184